nuxeo-features/nuxeo-admin-center/nuxeo-connect-update/src/main/java/org/nuxeo/connect/update/task/live/commands/Deploy.java

```diff
@@ -52,6 +58,10 @@ public Deploy(File file) {
         super(file);
     }
 
+    /**
+     * @since 9.3, reload mechanism has changed, keep it for backward compatibility
+     */
+    @Deprecated
```

Comment:
since annotations are confusing here, why not use "@ deprecated since 9.3 etc..."

Reply:
Yeah you're right. Thx